### PR TITLE
GUVNOR-3237: Change the home page to the new one

### PR DESCRIPTION
**Pull request by pefernan**
@mbarkley @manstis adding some missing changes to fix the build.

### CODE REVIEW

**Comment by manstis** [comment]
I'll defer to @paulovmr

**Comment by paulovmr** [comment]
@pefernan Sorry, I assumed all repositories were ok based on j-enkins full downstream b-uild. Is this repository getting built on it?

**Comment by pefernan** [comment]
@paulovmr no problem. It is not on the downstream build...

**Comment by alessioscalici** [comment]
@pefernan nice :)

**Comment by pefernan** [comment]
@mbarkley well seen! done

**Comment by mbarkley** [comment]
@pefernan Okay. I am still building locally to see if everything works now.

**Comment by mbarkley** [comment]
@pefernan I think also that we should be using UberFire 2.0.0-SNAPSHOT. I am trying that out locally now.

File: appformer-deps/pom.xml [hunk 1]
```diff
@@ -32,9 +32,9 @@
 
   <properties>
     <version.org.jboss.integration-platform>6.0.0.Final</version.org.jboss.integration-platform>
-    <version.org.uberfire>1.1.0-SNAPSHOT</version.org.uberfire>
+    <version.org.uberfire>1.2.0-SNAPSHOT</version.org.uberfire>
```
Comment (mbarkley) [inline appformer-deps/pom.xml]: I've confirmed locally that this should be `2.0.0-SNAPSHOT`.

**Comment by pefernan** [comment]
@mbarkley you're right... here you have. One day we should discuss if it is worth sustaining this module

**Comment by mbarkley** [comment]
@pefernan I plan to start helping @porcelli with the AF migration soon. As soon as we can move code from here to the new AF repo and delete this, I will gladly do so :)

**Comment by pefernan** [comment]
jenkins retest this

**Comment by pefernan** [comment]
@manstis could you please merge this?

**Comment by pefernan** [comment]
@mbarkley let me know when are you planning to do that please

**Pull request merged**